### ISSUE TITLE
Ссылка со старой страницы курсов

### ISSUE THREAD

**Issue by iliakan** [iliakan/javascript-nodejs#619]
Есть старый урл http://javascript.ru/courses.

Нужна какая-то плашка/ссылка, которая позволит мягко перенаправить людей на новый движок.

Убить страницу и сделать 301 нельзя, т.к. на новом движке Яндекс сейчас сходит с ума из-за переезда http -> https (поисковый трафик упал в 2-3 раза), а может и по другим причинам, ведь сайт сильно поменялся. Страница при этом просто пропадёт из результатов.

**Comment by Bezart** [comment]
По-моему мы раньше договаривались показывать уведомления на всю страницу. Что-то типа http://take.ms/nQKXt

**Comment by iliakan** [comment]
Нет, ты делал внутри страницы дизайн, вот http://javascript.ru/tutorial/foundation
Так же?

**Comment by Bezart** [comment]
Я бы все-таки сделал на всю страницу, как гугл делает. Но если есть технические нюансы, можно и так.

**Comment by iliakan** [comment]
А старые ссылки тоже изменить?

**Comment by Bezart** [comment]
Да, так было бы еще лучше
